app/models/user.rb

```diff
@@ -1,2 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Great job on including User model in devise however it would be better if you make it `:confirmable `and `:trackable`. `Confirmable `is responsible to verify if an account is already confirmed to. It adds an extra layer of security to the login.  The `trackable module` keeps track of the user by storing the sign in time, sign in count and IP address information. Trackable enhances the user experience. Imagine  you have to remember passwords and login credential of every single app you log into. Trackable saves users trouble of remembering there logs.